.gitconfig

```diff
@@ -39,32 +64,38 @@
 	bra = branch -a
 	st = status -sb
 	lg = log --oneline
-	gr = log --graph \
-	--pretty=format:'%C(bold yellow)%d\
-	%C(reset)%C(cyan)%h %C(dim white)-\
-	%C(reset)%s %C(dim blue)(%cr)\
-	%C(dim white)<%an>%C(reset)' \
-	--abbrev-commit --date=short
+	# If you want this in one line, then you can predefine "git-pretty"
+	# formattings! Check it out.
+	# Also, I didn't know the difference between author and committer until
+	# just now. Nice! Imma use authoer instead, like you. I think that's
+	# much better
+	# (https://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git)
+	gr = log --graph --pretty=chrislog --abbrev-commit --date=short
 	graph = !git gr --all
 
 	ci = commit
 	acm = !git add . && git commit -m
+	# "git acm" is functionally identical to "git cam" (right?)
 	cm = commit -m
 	cam = commit -am
 	amend = commit --amend
 	cane = commit --ammend --no-edit
 
 	branches = branch -a
+	# looool this is alias is troll
```

Comment:
I rarely use these, but I got them after reading some rants about git and then landing on this:
https://gggritso.com/human-git-aliases